ci_framework/roles/sosreport/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Ensure sos is installed
+  ansible.builtin.yum:
+    name: sos
+    state: present
+  become: yes
```

Comment:
please use `become: true` instead. And it's usually better to push it right after the `name: ` key. ansible-lint will probably complain about the boolean